src/main/java/ticTacToe/game/Mark.java

```diff
@@ -2,18 +2,36 @@
 
 public enum Mark {
 
-    NOUGHT("O"), CROSS("X");
+    NOUGHT("O"),
+    CROSS("X");
 
-    Mark(String name) {
-        mark = name;
+    Mark(String mark) {
+        sign = mark;
    }
 
-    public String mark;
+    public final String sign;
 
-    public static String switchPlayerMark(String currentMark) {
-        if (currentMark.equals(CROSS.mark)) {
-            return NOUGHT.mark;
+    public static boolean isValidMark(String markChosen) {
+        for (Mark mark : Mark.values()) {
```

Comment:
Don't you think it's overkill to have a loop here, as you only have two values?

This for example would have the same result and is shorter to read
`return markChosen == Mark.CROSS || markChosen == Mark.NOUGHT`

Reply:
Well, I'd need to change a lot of methods... :)

Reply:
You're right about the changes needed. It's a trade-off between flexibility and the changes needed to support that flexibility.

For this case, though, I'd think about how likely it is to get a third or a fourth or a fifth `Mark` for a tic tac toe game.

Reply:
Sure...